### PR TITLE
update deps to chef-13/ohai-13

### PR DESCRIPTION
**Pull request by lamont-granquist**
release blocker for chef-dk 2.0

### CODE REVIEW

File: opscode-pushy-client.gemspec [hunk 1]
```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = PushyClient::VERSION
 
-  gem.add_dependency "chef", "~> 12.19" # pin to latest pre-13
-  gem.add_dependency "ohai", "~> 8.23" # pin to latest pre-13
+  gem.add_dependency "chef", "~> 13.0"
```
Comment (thommay) [inline opscode-pushy-client.gemspec]: should this be >= 12 < 14 or something?

Comment (lamont-granquist) [inline opscode-pushy-client.gemspec]: IDK?  travis seems angry maybe there's an answer there...

**Comment by tduffield** [comment]
What type of version bump should be included with this? I wonder if we need a MAJOR version bump.

**Comment by tduffield** [comment]
We'll also need to update the Travis stuff to mirror the Ruby versions for Chef 13 (2.3.X and 2.4.X probably)

**Comment by lamont-granquist** [comment]
the external API of pushy isn't really changing though?

**Comment by lamont-granquist** [comment]
@here okay now we're still allowing chef-12, i think that makes this definitely a minor bump?

**Pull request merged**